tests/test_sequence_labeling_encoder.py

```diff
@@ -65,3 +67,24 @@ def test_sequence_label_encoder_half():
     )
     label_arr = encoder.transform(out, labels)
     assert label_arr == [0, 1, 0]
+
+def test_sequence_label_bio_tagging():
+    encoder = SequenceLabelingEncoder(pad_token="<PAD>", bio_tagging=True)
+    labels = [
+        {'start': 0, 'end': 4, 'label': 'z', 'text': 'five'},
+        {'start': 5, 'end': 12, 'label': 'z', 'text': 'percent'},
+        {'start': 13, 'end': 17, 'label': 'z', 'text': '(5%)'},
+    ]
+    encoder.fit([labels])
+    assert len(encoder.classes_) == 3
+
+    out = EncodedOutput(
+        token_ids=np.array([   0, 9583,  139,   40,  249, 8875,    2]), 
+        tokens=np.array(['0', 'five', ' per', 'cent', ' (', '5', '%)', '2'], dtype='<U21'), 
+        token_ends=np.array([-1,  4, 8, 12, 14, 15, 17, -1]), 
+        token_starts=np.array([-1,  0,  5, 8, 13, 14, 15, -1]), 
+        useful_start=0, 
+        useful_end=512
+    )
+    label_arr = encoder.transform(out, labels)
+    assert label_arr == [0, 1, 1, 2, 1, 2, 1, 0]
```

Comment:
I'm struggling to see how this is the expected output for BIO tagging.  Assuming 1 represents "B", where is the third 1 coming from?

Reply:
Test was broken, thanks for the catch.